### PR TITLE
Update array_ops

### PR DESCRIPTION
**Pull request by jvishnuvardhan**
Updated tf.identity with an example.

### CODE REVIEW

File: tensorflow/python/ops/array_ops.py [hunk 1]
```diff
@@ -58,8 +58,20 @@
...
 @dispatch.add_dispatch_support
 def identity(input, name=None):  # pylint: disable=redefined-builtin
-  r"""Return a tensor with the same shape and contents as input.
+  r"""Identity returns the value of the input tensor at the time of evaluation.
 
+  For example:
+
+  ```python
+  import tensorflow as tf
+  with tf.device('cpu:0'):
```
Comment (alextp) [inline tensorflow/python/ops/array_ops.py]: You don't need the device block

**Review by alextp — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/array_ops.py (hunk 1 above)
Comment (jvishnuvardhan) [reply tensorflow/python/ops/array_ops.py]: Updated. Thanks Alex.

**Review by alextp — APPROVED** [review]

**Comment by jvishnuvardhan** [comment]
@yifeif Can you please merge this to r2.0. Thanks!

**Comment by rthadur** [comment]
Closing this PR as this not against `master`, please open a new PR against `master` 
CC @mihaimaruseac

**Pull request closed without merging**